config/params.ini

```diff
@@ -75,7 +75,9 @@ belt_rejuv_columns=2
 belt_hp_columns=1
 belt_mp_columns=1
 pre_buff_every_run=0
-
+#skizot
```

Comment:
comments about user information of changes can be removed. That's why we have git : )

Reply:
lol it's an OLD habit.